Cargo.toml

```diff
@@ -50,6 +50,10 @@ fuchsia-zircon = { version = "0.3.2", optional = true }
 # use with `--target wasm32-unknown-unknown --features=stdweb`
 stdweb = { version = "0.4", optional = true }
 
+[target.wasm32-unknown-emscripten.dependencies]
```

Comment:
This completely ignores the existence of the asmjs target or the experimental target.